src/renderers/shaders/ShaderChunk/morphtarget_pars_vertex.glsl.js

```diff
@@ -20,12 +20,34 @@ export default /* glsl */`
 
 		vec4 getMorph( const in int vertexIndex, const in int morphTargetIndex, const in int offset ) {
 
-			int texelIndex = vertexIndex * MORPHTARGETS_TEXTURE_STRIDE + offset;
+			int texelIndex = vertexIndex * MORPHTARGETS_TEXTURE_STRIDE + 3 * offset;
+
 			int y = texelIndex / morphTargetsTextureSize.x;
 			int x = texelIndex - y * morphTargetsTextureSize.x;
 
 			ivec3 morphUV = ivec3( x, y, morphTargetIndex );
-			return texelFetch( morphTargetsTexture, morphUV, 0 );
```

Comment:
Are multiple fetches really faster than three or even four? I'm afraid this needs to be tested one more than one devices. Especially mobile devices could show a performance regression here.

Reply:
As it is currently works with rgba format it will always fetch 4 floats so fetching 3 instead should always be faster especially on mobiles where bandwidth is precious. I'll replace the horse example with the faces one and do some more testing but I only have one android phone with and older snapdragon and the quest 3 which all show similar improvements

Reply:
Alternatively, you could also add a slider to `webgl_instancing_morph` that controls the number of instances.

Reply:
I would prefer to have the faces instead because they are using more influences and also morph the normals so its a more complete case than the horses. It's also heavier so more useful to test things like performance in the future